simulator/src/main/java/com/hazelcast/simulator/utils/PropertyBindingSupport.java

```diff
@@ -301,6 +326,10 @@ private static Field findPropertyField(Class clazz, String property) {
         }
     }
 
+    private static void throwBindException(String msg, Object... args) {
```

Comment:
I just wanted to have shorter lines in the code, so the exceptions fit into a single line. I'll have a look if this is still necessary with the actual indentation.